extensions/amp-consent/amp-consent.md

```diff
@@ -291,7 +290,29 @@ Individual components may override this behavior to provide more specialized han
 </amp-analytics>
 ```
 
-### Advanced Consent Blocking Behaviors
+### Advanced Pre-defined Consent Blocking Behaviors
+AMP provides a list of pre-defined [consent policy instances](#policy-instance-optional) for publishers to easily define consent blocking behaviors to individual components.
+
+Set the value to the `data-block-on-consent` attribute to use the pre-defined consent blocking behavior policy.
+
+*Example: Blocking the analytics until user respond to consent*
+
+```html
+<amp-analytics data-block-on-consent="_till_responded"
+  type="googleanalytics">
+</amp-analytics>
+```
+
+AMP may support more advanced pre-defined blocking behaviors in the future. Because of this, the value of `data-block-on-consent` is reserved only for the following supported pre-defined attributes.
+
+The supported pre-defined attributes are
+* `_till_responded` : Unblock the component until the user has responded to the consent prompt, or the consent prompt has been skipped.
+* `_till_accepted` : [Default basic blocking behavior](#basic-blocking-behaviors), expect that when `_till_accepted` is explicitly added, individual components cannot override the blocking behavior.
+* `_auto_reject` : Always reject the consent automatically if consent is required but unknown. The reject consent decision will not be stored. It is recommended not to specify a consent prompt UI when auto rejecting consent for every components.
+
+When one of the pre-defined attributes is used, AMP assumes that the publisher takes final control on the consent blocking behaviors. Individual components cannot override the blocking behaviors brought by pre-defined consent policy, they can however still customize components' behaviors after having been unblocked.
```

Comment:
Shall we add a section somewhere earlier on the page to explain this at higher level:

Consent handling is a 2-stage serial process 
- Stage 1 is pub control to block elements from instantiation using data-block-on-consent attribute.
- Stage 2 is each component's control 

Pub can leave policy blank to have component to take control over stage 1.

Not the exact wording, but something like that explained at high level should be useful for users to understand.

Reply:
I'll address in separate PR.